app/services/solidus_pay_tomorrow/base_service.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module SolidusPayTomorrow
+  class BaseService
+    def initialize(**_args) end
```

Comment:
Considering you added BaseService, what do you think to let to Client::BaseService to inherit from this Object?

ref https://github.com/nebulab/solidus_pay_tomorrow/blob/301245cdf773310a08d8d373683b644f3a5da76c/app/services/solidus_pay_tomorrow/client/base_service.rb#L7-L24

Reply:
Yes, was thinking of doing it!
Will do before merging.

Reply:
Done in 272ffaf5f9a4cd678157d668abab6f88587d85a1